### DISCUSSION TITLE
Shouldn't the java11 branch be deleted?

### DISCUSSION THREAD

**Discussion by GitMensch** [fesch/Structorizer.Desktop#1168] [Ideas]
At least in this repo all changes done there are also in master and since https://github.com/fesch/Structorizer.Desktop/releases/tag/3.32-06 we require java 11.

**Comment by codemanyak** [comment]
Well, I did so now. Of course, this leads to some dead links (Error 404) in the project history, e.g. when inspecting the respective pull request. Therefore I am always a little hesitant to remove branches. But we won't certainly return to any pre-Java-11 version, and the commits are still retrievable.
